src/ChannelAllocationChromosomeMutation.java

```diff
@@ -0,0 +1,41 @@
+public class ChannelAllocationChromosomeMutation implements IMutationAlgorithm<ChannelAllocationChromosome> {
```

Comment:
```suggestion
public class ChannelAllocationChromosomeUniformMutation implements IMutationAlgorithm<ChannelAllocationChromosome> {
```
Change class name to follow naming convention and allow NonUniform implementation